alf/trainers/on_policy_trainer.py

```diff
@@ -115,15 +122,16 @@ def train_():
             if (iter + 1) % checkpoint_interval == 0:
                 checkpointer.save(global_step=global_step.numpy())
 
-            if (iter + 1) % eval_interval == 0:
+            if eval_env is not None and (iter + 1) % eval_interval == 0:
                 with tf.summary.record_if(True):
-                    eager_compute(eval_metrics, eval_env,
-                                  algorithm.predict_state_spec,
-                                  algorithm.greedy_predict,
-                                  num_eval_episodes,
-                                  global_step,
-                                  eval_summary_writer,
-                                  "Metrics")
+                    eager_compute(metrics=eval_metrics,
+                                  environment=eval_env,
+                                  state_spec=algorithm.predict_state_spec,
+                                  action_fn=algorithm.greedy_predict,
+                                  num_episodes=num_eval_episodes,
+                                  train_step=global_step,
+                                  summary_writer=eval_summary_writer,
+                                  summary_prefix="Eval/Metrics")
```

Comment:
Since the summary will be stored in different directory, it should be okey to just use summary_prefix="Metrics"?